### DISCUSSION TITLE
GitHub と InnerSource

### DISCUSSION THREAD

**Discussion by yuhattor** [InnerSourceCommons/jp-roadmap#36] [Meetup]
@yuichielectric
これ、何か今後コミュニティで話せそうなことありますか？？
InnerSource と、GitHub の設定〜
とかでもいいかもしれないし、もう少し GitHub における InnerSource みたいな話題でもいいかも↓みたいな感じです。
https://github.blog/2020-03-11-why-organizations-should-commit-to-innersource-in-2020/

いかがでしょうか？

**Comment by yuichielectric** [comment]
@yuhattor  前に服部さんと一緒にやったデブサミでの発表内容をアップデートしてとかですかね？
https://speakerdeck.com/yuichielectric/how-to-implement-innersource?slide=25

**Reply by yuhattor** [reply]
@yuichielectric  そうですね！
とくにツールのところの整理や、どうやって GitHub を含めたそれらのツーリングを使っていくのか、最初にどんなメトリクスをとるとよくて、ゆくゆくはどのようにメトリクスを取るのが良いのかなどがあればいいと思います。
例えば InnerSource Portal とかって、最初に "インナーソースのプロジェクト" がない時に立ち上げても大きな意味を持たないと思います。

どうやって体系的に GitHub や関連ツールを使っていったら良いのかなどもまとまりが欲しかったりします。
